starklings.py

```diff
@@ -1,24 +1,42 @@
 from argparse import ArgumentParser
 import asyncio
 from pathlib import Path
+from tokenize import String
 from src import cli
+from typing import List
```

Comment:
Not used neither

Reply:
We need a linter
Why don't we want one right now ?